inference-engine/src/gna_plugin/gna_plugin.cpp

```diff
@@ -754,12 +760,13 @@ void GNAPlugin::LoadNetwork(CNNNetwork & _network) {
 
     // network optimisation phases
     int passIdx = 0;
-    auto run_passes = [&] (const CNNNetwork& network, bool runBeforeCopy, bool lowPrecision) {
+    auto run_passes = [&] (const CNNNetwork& network, bool runBeforeCopy, bool lowPrecision, bool isNgraphPassesUsed) {
```

Comment:
Too complicated. `isNgraphPassesUsed` is available here without extending arguments list of the function.